httpx/config.py

```diff
@@ -251,8 +252,15 @@ def __init__(self, http_versions: HTTPVersionTypes = None):
             self.http_versions = {http_versions.upper()}
         elif isinstance(http_versions, HTTPVersionConfig):
             self.http_versions = http_versions.http_versions
+        elif isinstance(http_versions, Iterable):
+            self.http_versions = {
+                version.upper() if isinstance(version, str) else version
+                for version in http_versions
+            }
         else:
-            self.http_versions = {version.upper() for version in http_versions}
+            raise TypeError(
+                f"HTTP version should be a string or list of strings, but got {type(http_versions)}"
```

Comment:
You can break this into two strings to appease [flake8](https://travis-ci.org/encode/httpx/jobs/574508042#L214):
```suggestion
                "HTTP version should be a string or list of strings,"
                f" but got {type(http_versions)}"
```

Reply:
thanks, I overlooked flake8 error. if flake8 is not successful, nox can be stopped.